recipes/mpir/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "2.7.2" %}
+
+package:
+  name: mpir
+  version: {{ version }}
+
+source:
+  fn: mpir-{{ version }}.tar.bz2
+  url: http://mpir.org/mpir-{{ version }}.tar.bz2
+  md5: b31aa1ebc01b5fedf4ac43277c03feec
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - m4  # [unix]
+    - python
+    - vs2015_runtime # [win]
+  run:
+    - vs2015_runtime # [win]
+
+test:
+  files:
```

Comment:
This forces the selection magic to work. Basically CPython devs recommend each version major-minor version of CPython be built with one version of MSVC. So, Continuum followed these guidelines when building CPython. As a result, installing a particular version of CPython drags in some information about the MSVC to use. This will then select which version of a package would be used. Given MPIR's constraints, only Python 3.5 will work and will activate `vc14`, which drags in this package. Without this, the package will fail to install.